<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Frame</class>
 <widget class="QFrame" name="Frame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>126</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <property name="toolTip">
   <string>AIDE...</string>
  </property>
  <property name="frameShape">
   <enum>QFrame::StyledPanel</enum>
  </property>
  <property name="frameShadow">
   <enum>QFrame::Raised</enum>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>224</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="0">
     <widget class="QSpinBox" name="spinBox_x"/>
    </item>
    <item row="2" column="1">
     <widget class="QSpinBox" name="spinBox_y"/>
    </item>
    <item row="2" column="2">
     <widget class="QSpinBox" name="spinBox_z"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Angle (x)</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="angle_y">
      <property name="text">
       <string>Angle (y)</string>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLCDNumber" name="lcdNumber_z"/>
    </item>
    <item row="1" column="1">
     <widget class="QLCDNumber" name="lcdNumber_y"/>
    </item>
    <item row="1" column="0">
     <widget class="QLCDNumber" name="lcdNumber_x"/>
    </item>
    <item row="3" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Angle (z)</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_start">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Start le timer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_stop">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>70</y>
     <width>114</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop le timer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>spinBox_x</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_x</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>63</x>
     <y>91</y>
    </hint>
    <hint type="destinationlabel">
     <x>67</x>
     <y>56</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_y</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_y</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>86</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBox_z</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_z</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>226</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
